pom.xml

```diff
@@ -606,6 +606,12 @@
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
             <version>${version.com.puppycrawl.tools.checkstyle}</version>
+            <exclusions>
```

Comment:
It is actually strange that it fails, because they have that `com.sun:tools` dependency packed in a dedicated profile https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L1167-L1184 that should be activated only on Oracle JDKs. I have no idea what is wrong. Anyway, excluding `com.sun:tools` here should be fine, because -- I strogly hope -- we do not have any checks depending on it.